### DISCUSSION TITLE
mo.ui.altair_chart item hint possible without selection?

### DISCUSSION THREAD

**Discussion by f-fuchs** [marimo-team/marimo#2473] [Q&A] [answered]
hey,

I am new to marimo and I was wondering if it is possible to get the item hint without allowing selection?
![image](https://github.com/user-attachments/assets/412c58b7-a750-4532-811d-1ffcb5865cd1)

More concretely my problem is that I have a chart where the color fades as soon as I zoom in. 

original plot
![image](https://github.com/user-attachments/assets/a5018edc-30f8-4889-83a4-6aca92d964a5)

zoomed in plot
![image](https://github.com/user-attachments/assets/71116dd3-f317-456f-9e56-3b16cf750399)

code:
```
archetypes_training_size = pl.DataFrame({"BackboneArchetype": ["CONVNEXT", "CAFORMER"]})
archetypes_training_size = mo.ui.table(archetypes_training_size, selection="multi")
```
```
_df = experiments_label_noise_training_size.filter(
    pl.col("BackboneArchetype").is_in(archetypes_training_size.value.select(pl.col("BackboneArchetype")))
)
chart_training_size = mo.ui.altair_chart(
    alt.Chart(_df)
    .mark_point(opacity=1)
    .encode(
        x=alt.X("Backbone:O", sort=alt.EncodingSortField(op="sum", field="#Params")),
        y="PSNR:Q",
        color=alt.Color("BackboneArchetype:N"),
        size="Train Percentage:Q",
        shape="Training Strategy"
    )
    .properties(width=800, height=800)
    .interactive(),
    # disable automatic selection
    chart_selection=True,
    legend_selection=False
)
mo.vstack([archetypes_training_size, chart_training_size])
```
I think I narrowed the problem down to the chart_selection, when I disable it the opacity stays the same. The problem is the item hint also goes away. Is there  a way to keep the item hint without having everything fade (look washed out)?

**Comment by mscolnick** [comment, marked as answer]
You can just use Altair without `mo.ui.altair_chart` - and configure tooltips: https://altair-viz.github.io/gallery/scatter_tooltips.html

**Reply by f-fuchs** [reply]
thx 👍
